frontend/.claude/skills/e2e-tester/SKILL.md

```diff
@@ -26,6 +26,7 @@ Write end-to-end tests using Playwright against a full Redpanda Console stack ru
 - Test UI component rendering (that belongs in unit/integration tests)
 - Use brittle CSS selectors like `.class-name` or `#id`
 - Use force:true when calling .click()
+- Use waitForTimeout in e2e tests
```

Comment:
I would suggest to 1st attempt to wait for some element, or URL, or network request instead before falling back to the timeout.